index.html

```diff
@@ -1,61 +1,196 @@
+              <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+            </ul>
+          </div>
+          <button type="button">See project</button>
         </div>
-      </section>
-    </main>
-  </body>
+      </div>
+      <div class="card">
+        <img class="img-project" src="/img/Snapshoot-Portfolio4.svg" alt="">
+        <div class="description-project multi">
+          <h3>Multi-Post Stories</h3>
+          <div class="frame">
+            <p>CANOPY</p>
+            <img src="/img/Counter.svg">
+            <p>Back End Dev</p>
+            <img src="/img/Counter.svg">
+            <p>2015</p>
+          </div>
+          <p class="text-description">A daily selection of privately personalized reads; no accounts or sign-ups
+            required.</p>
+          <div class="technologys">
+            <ul class="list-technologys">
+              <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+              <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+              <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+            </ul>
+          </div>
+          <button type="button">See project</button>
+        </div>
+      </div>
+    </section>
+    <section class="section-p3" id="about">
+      <div class="description1">
+        <h2>About Myself</h2>
+        <p>
+          I'm a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
+        <p class="connect">lest's connect</p>
+      </div>
+      <div class="description2">
+        <ul class="social-container2">
+          <li><a href="https://github.com/Kal2403"><img src="/img/github.svg"/></a></li>
+          <li><a href="https://www.linkedin.com/in/cristhian-castillo-41136322a/"><img src="/img/Linkedin.svg"/></a></li>
+          <li><a href="#"><img src="/img/thougn.svg" alt="" /></a></li>
+          <li><a href="https://twitter.com/Kal24031992"><img src="/img/Twitter.svg"/></a></li>
+          <li><a href="#"><img src="/img/M.svg" alt="" /></a></li>
+        </ul>
+        <button type="button">Get my resume</button>
+      </div>
+      <div class="languages">
+        <ul class="name-drop">
+          <li>
+            <a href="#">
+              <p>Lenguages</p>
+              <img src="/img/Disabled.svg">
+            </a>
+          </li>
+          <img class="line" src="/img/Separator-Bottom.svg">
+          <li>
+            <a href="#">
+              <p>Frameworks</p>
+              <img src="/img/Disabled.svg">
+            </a>
+          </li>
+          <img class="line" src="/img/Separator-Bottom.svg">
+          <li>
+            <a href="#">
+              <p>Skills</p>
```

Comment:
- [ ] [Optional] Its very important to stick to figma design template according to the project requrement but but as the previous reviewer ignore  didnt request it for you to implement ,Please i would suggest you to implemt the language images just like how it look like on figma you will still need the section on this project along the line.please fix it .  
- [ ]  figma design 
![figma real](https://user-images.githubusercontent.com/92458236/235272570-c8761522-fbd5-47b1-8dee-50f316092947.jpg)

- [ ] your design 
![you d](https://user-images.githubusercontent.com/92458236/235272609-fe4b7982-bfe5-4a3a-ba5f-bb0c4b2ec607.jpg)